apps/api/src/app/workflows-v2/shared/map-step-type-to-result.mapper.ts

```diff
@@ -1,11 +1,51 @@
 import { ActionStepEnum, actionStepSchemas, ChannelStepEnum, channelStepSchemas } from '@novu/framework/internal';
+import { JSONSchema } from 'json-schema-to-ts';
+import { StepTypeEnum } from '@novu/shared';
 
-export const mapStepTypeToResult = {
-  [ChannelStepEnum.SMS]: channelStepSchemas[ChannelStepEnum.SMS].result,
-  [ChannelStepEnum.EMAIL]: channelStepSchemas[ChannelStepEnum.EMAIL].result,
-  [ChannelStepEnum.PUSH]: channelStepSchemas[ChannelStepEnum.PUSH].result,
-  [ChannelStepEnum.CHAT]: channelStepSchemas[ChannelStepEnum.CHAT].result,
-  [ChannelStepEnum.IN_APP]: channelStepSchemas[ChannelStepEnum.IN_APP].result,
-  [ActionStepEnum.DELAY]: actionStepSchemas[ActionStepEnum.DELAY].result,
-  [ActionStepEnum.DIGEST]: actionStepSchemas[ActionStepEnum.DIGEST].result,
-};
+export function computeResultSchema(stepType: StepTypeEnum, payloadSchema?: JSONSchema) {
+  console.log('computeResultSchema:stepType', stepType);
+  console.log('computeResultSchema:payloadSchema', payloadSchema);
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** remove logs‏